open_in_mpv/open_in_mpv.py

```diff
@@ -133,53 +173,39 @@ def real_main(log: TextIO) -> int:
     message. Then the message is expected to be proceed.
     """
     os.makedirs(dirname(MPV_SOCKET), exist_ok=True)
-    stdin_buffer = sys.stdin.buffer
-    req_len = struct.unpack('@i', stdin_buffer.read(4))[0]
-    message = json.loads(stdin_buffer.read(req_len).decode())
-    logger.debug(f'Message contents ({req_len}): {message}')
-    if 'init' in message:
-        resp = json.dumps(dict(version=VERSION, logPath=log.name, socketPath=MPV_SOCKET)).encode()
-        size = struct.pack('@i', len(resp))
-        stdout_buffer = sys.stdout.buffer
-        stdout_buffer.write(size)
-        stdout_buffer.write(resp)
+    message: Dict[str, Any] = request(sys.stdin.buffer)
+
+    if message['init']:
+        response(dict(version=VERSION, logPath=log.name, socketPath=MPV_SOCKET))
         log.close()
         return 0
-    try:
-        url: str = message['url']
-    except KeyError:
+
+    if ((url := message.get('url', None)) is None): # pylint: disable=C0325
```

Comment:
Don't disable C0325 here. Just remove the extra parentheses. Only the parentheses around the assignment are required.

Reply:
This makes pylint then believe it is passing a Literal[bool] over passing a string.

Reply:
Okay leave the disable but remove the extra set of parens.

Reply:
Doing so will result in two more ignores; as long as you don't mind that alright they will be removed.

Reply:
actually only one; that disable was able to be removed once the extra parentheses were removed.